### PR TITLE
Yep 62 token storage system

### PR DESCRIPTION
**Pull request by filveith**

### CODE REVIEW

**Comment by ASparton** [comment]
Why are all the NavBar changes mentioned in this PR?

**Review by ASparton — COMMENTED** [review]
What are the advantages of using Redux here? Can't we just have a TypeScript class that we can import whenever we want inside a component?

**Comment by filveith** [comment]
> 

No idea :))

File: front/package.json [hunk 1]
```diff
@@ -18,17 +18,20 @@
     "@emotion/react": "^11.11.1",
     "@ionic/react": "^7.0.0",
     "@ionic/react-router": "^7.0.0",
+    "@ionic/storage": "^4.0.0",
     "@mantine/carousel": "^6.0.13",
     "@mantine/core": "^6.0.13",
     "@mantine/form": "^6.0.13",
     "@mantine/hooks": "^6.0.13",
+    "@reduxjs/toolkit": "^1.9.5",
```
Comment (ASparton) [inline front/package.json:26]: Uninstall?

File: front/src/main.tsx [hunk 2]
```diff
@@ -1,54 +1,58 @@
 import { MantineProvider } from '@mantine/core';
 import { createRoot } from 'react-dom/client';
 import App from './App';
+import { Provider } from 'react-redux';
```
Comment (ASparton) [inline front/src/main.tsx:4]: Import not needed anymore

**Review by ASparton — COMMENTED** [review]
Just need to clean up redux

**Review by ASparton — APPROVED** [review]

**Pull request merged**